parent/pom.xml

```diff
@@ -1125,7 +1125,7 @@
                 <!-- temporary forms SDK -->
                 <groupId>com.adobe.aemfd</groupId>
                 <artifactId>aem-forms-sdk-api</artifactId>
-                <version>1.0.0-52cdda2</version>
+                <version>2022.08.29.00</version>
```

Comment:
This is not required

Reply:
Just saw nitins comments, please change the SDK version to latest public release.